mizar/daemon/interface_service.py

```diff
@@ -212,8 +212,9 @@ def ConsumeInterfaces(self, request, context):
                 if requested_pod_name in self.pod_dict:
                     if self.pod_dict[requested_pod_name]:
                         # Interfaces for the Pod has been produced
+                        interfaces = self._ConsumeInterfaces(requested_pod_name, request)
```

Comment:
I tried this patch and found that VPC & subnets are stuck in Init state. Did you see the same outcome?

Reply:
I tried a lot in both local and kube-up, not seeing that. And this simple change is unlikely cause that.